lib/template-mixins.js

```diff
@@ -84,22 +84,30 @@ module.exports = function (fields, options) {
             return translate(sharedTranslationsKey + key);
         };
 
-        var conditionaltranslate = function (key) {
+        // Like t() but returns null on failed translations
+        var conditionalTranslate = function (key) {
             key = sharedTranslationsKey + key;
             var translated = translate(key);
             return translated !== key ? translated : null;
         };
 
+        var getTranslationKey = function (key, property) {
+            return fields && fields[key] ? fields[key][property] : 'fields.' + key + '.' + property;
```

Comment:
This assumes a label/hint is set for each listed field. If these aren't set the `label` is returned as `undefined` and the `hint` doesn't show at all.

I think it's probably worth padding out the tests to cover all the use cases. For labels, it's currently possible to set `label: "My label"` for input text fields.